### DISCUSSION TITLE
Enhancements to OSSI with Breaking Changes

### DISCUSSION THREAD

**Discussion by iamrahul127** [DependencyTrack/dependency-track#1458] [General]
Received email form Sonatype as below. Will this impact DT? 

![image](https://user-images.githubusercontent.com/24877322/157387481-986f15cc-1025-4f61-a118-7bd1d7f151fe.png)

**Comment by stevespringett** [comment]
Duplicate of https://owasp.slack.com/archives/C6R3R32H4/p1646754001687379

<img width="374" alt="Screen Shot 2022-03-09 at 12 54 30 AM" src="https://user-images.githubusercontent.com/3878933/157388433-1ab79c08-922c-4826-8cba-b7c131134f4e.png">
